app/scripts/modules/core/src/domain/IManagedEntity.ts

```diff
@@ -103,10 +103,35 @@ export interface IManagedArtifactVersion {
     statelessConstraints?: IStatelessConstraint[];
   }>;
   build?: {
-    id: number;
+    id: number; // deprecated, use number
+    number: string;
+    uid: string;
+    job: {
+      link: string;
+      name: string;
+    };
+    startedAt: string;
+    completedAt?: string;
+    status: 'SUCCESS' | 'UNSTABLE' | 'BUILDING' | 'ABORTED' | 'FAILURE' | 'NOT_BUILT';
```

Comment:
@gal-yardeni can you confirm that these are the actual values I might get? I guessed these from reading some of your PRs, but I don't actually see them spelled out in the `BuildMetadata` class in keel.

Reply:
sure -- I added this yesterday and the values can be:
```
enum class CompletionStatus {

  INCOMPLETE,
  SUCCEEDED,
  FAILED,
  ABORTED

Reply:
you are corret! 
```
enum class Result {
  SUCCESS, UNSTABLE, BUILDING, ABORTED, FAILURE, NOT_BUILT
}

Reply:
nice thank you 🙏